cli/deployment/flags.go

```diff
@@ -0,0 +1,447 @@
+package deployment
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/coreos/go-oidc/v3/oidc"
+	"github.com/spf13/pflag"
+
+	"github.com/coder/coder/cli/cliflag"
+	"github.com/coder/coder/codersdk"
+)
+
+const (
+	secretValue = "********"
```

Comment:
Should we have some way to indicate that the value is sensitive in the flag type? I'd hate for the FE to have to check for this string to see if it's sensitive.

Reply:
I had a sensitive field originally but because you can't cleanly loop the struct fields it didn't make sense to go that route for me. I can see a use case for a sensitive field so the frontend can do logic though, so I think this is a good suggestion to add it back.

Reply:
I've made the change to add a `Secret bool` field and we use reflect to ensure all the secret values are automatically scrubbed before being returned from the API.

Reply:
That seems nice to me